playbook/app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -16,19 +16,26 @@ class Title < Playbook::KitBase
                      default: nil,
                      deprecated: true
       prop :bold, type: Playbook::Props::Boolean, default: true
+      prop :truncate, type: Playbook::Props::Enum,
+                      values: [nil, "1", "2", "3", "4", "5"],
+                      default: nil
 
       def classname
         if is_size_responsive
-          generate_classname("pb_title_kit", variant, color, is_bold) + generate_responsive_size_classname
+          generate_classname("pb_title_kit", variant, color, is_bold, is_truncate) + generate_responsive_size_classname
         else
-          generate_classname("pb_title_kit", size, variant, color, is_bold)
+          generate_classname("pb_title_kit", size, variant, color, is_bold, is_truncate)
         end
       end
 
       def is_bold
         bold ? nil : "thin"
       end
 
+      def is_truncate
+        truncate ? "truncate_#{truncate}" : nil
+      end
```

Comment:
If we're extending kits we need to add a doc and also make sure React is covered.